config/manifests/bases/backstage-operator.clusterserviceversion.yaml

```diff
@@ -17,7 +17,11 @@ spec:
       kind: Backstage
       name: backstages.rhdh.redhat.com
       version: v1alpha1
-  description: Operator to deploy Backstage on Kubernetes
+  description: |
+    Operator to deploy Backstage on Kubernetes
+
+    The telemetry data collection feature is enabled by default. Red Hat Developer Hub sends telemetry data to Red Hat by using the `backstage-plugin-analytics-provider-segment` plugin. To disable this and to learn what data is being collected, see https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.2/html-single/administration_guide_for_red_hat_developer_hub/index#assembly-rhdh-telemetry_admin-rhdh
```

Comment:
same comment as above -- see upstream doc in https://github.com/janus-idp/backstage-showcase/pull/1028/files#diff-8b34298b8105f19a866e1072c43ff5fa2675f4b29a8511a74e6152c88a171718R9-R29 ... no mention of RHDH in the upstream.

Reply:
I would prefer to move operator part from showcase document to operator repo (as TELEMETRY.md or so) and maintain it here. The reason is that we are evolving and I'd like to make sure the info is updated.
The message should be like - default Backstage operator configuration (which is RHDH) contains `backstage-plugin-analytics-provider-segment` dynamic plugin which sends telemetry data to Red Hat. To disable this and to learn what data is being collected <TELEMETRY.md>

Reply:
The operator itself doesn't collect any telemetry; it is RHDH/JanusShowcase that does that. Keeping separate TELEMETRY.md in this repo might be confusing, and it would definitely create a duplication as this information is already managed in [Janus Showcase dosc](https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/getting-started.md#telemetry-collection) and RHDH docs

Reply:
We can link to the upstream  docs (https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/getting-started.md#telemetry-collection) here, but we need to make sure that the during RHDH build it is updated to the downstream docs.


We can also reword it so it doesn't mention the product name to make it easier to update.

> The telemetry data collection feature is enabled by default. The default configuration uses image with `backstage-plugin-analytics-provider-segment` plugin enabled and configured. To disable this and to learn what data is being collected, see ... 

link for upstream (this repo): 
https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/getting-started.md#telemetry-collection
link for downstream (RHDH build):
https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.2/html-single/administration_guide_for_red_hat_developer_hub/index#assembly-rhdh-telemetry_admin-rhdh

Reply:
I've reworded it and also updated the link to the one mentioned for upstream